content/work/estimating-engine.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Estimating Engine
+sub: Create & manage custom manufacturing quotes
+logo:
+image:
+  src: work/estimating-engine/estimating-engine.jpg
+  alt: manufacturing quote details
+  position: top
+client: &client General Stamping & Metalworks
+date: 2020-01-20
+end: 2021-12-01
+tags:
+  - Manufacturing Sector
+people:
+  - &ryan
+    name: Ryan Hochstetler
+    face: james-weber.jpg
+    url: https://www.linkedin.com/in/ryanhochstetler/
+    title: Information Systems Administrator
+    venue: *client
+press:
+  - text: |
+      @@@
+    <<: *ryan
+    slug: configurations
+summary: |
+  OddBird helped General Stamping & Metalworks
+  plan, design, and develop Estimating Engine,
+  a responive web application.
+  Estimating Engine enables estimators at GSM 
+  to efficiently 
+  create and manage custom quotes 
+  to manufacture parts
+  for their customers. 
+---
+
+{% import 'embed.macros.njk' as embed %}
+{% import 'layout.macros.njk' as layout %}
+{% import 'quotes.macros.njk' as quotes %}
+
+
+{{ layout.title('What We Did') }}
+
+{% call embed.media_block(
+  media=embed.img(
+    src='work/estimating-engine/ee-example.jpg',
+    alt='details of part for manufacturing quote'
+  ),
+    name='desktop-work'
+) %}
+
+{% call layout.grid('narrow-columns') %}
+
+{% call layout.block('column') %}
+
+### Services
+
+  - Research & Concepting
+  - Project Management
+  - UX & UI Design
+  - Front-end Development
+  - Back-end Development
+  - Ongoing Maintenance
+  - Design System
+
+
+
+{% endcall %}
+
+{% call layout.block('column') %}
+
+### Languages
+
+  - Adobe XD
+  - Django & Python
+  - Herman
+  - JavaScript with Vue.js
+  - Sass & CSS
+
+
+
+{% endcall %}
+
+{% endcall %}
+
+{% endcall %}
+
+{{ quotes.grid(press) }}
```

Comment:
The pattern in our other projects has been to put quotes above that 'what wee did' -- I think we may want to be consistent on that (unless we're actually removing this quote entirely?)

Reply:
See comment above.